README.md

```diff
@@ -81,4 +152,4 @@ I would like to thank
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] The MIT license link is broken in the README file, Please make sure that you fix it.

![image](https://user-images.githubusercontent.com/83803414/208891653-4ed72fe5-f69b-4903-9298-d61bf03f3c75.png)